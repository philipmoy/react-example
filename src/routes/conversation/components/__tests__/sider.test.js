import React from 'react';
import { shallow } from 'enzyme';

import Sider from '../sider';

describe('ConversationSider', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <Sider />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
