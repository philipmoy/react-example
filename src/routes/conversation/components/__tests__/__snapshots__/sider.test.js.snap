// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConversationSider renders correctly 1`] = `
<div
  className="conversation-sider"
>
  <button
    className="btn btn--exit"
    type="button"
  >
    EXIT
  </button>
  <h1>
    This is a fun headline.
  </h1>
  <p>
    Intro copy that will help users navigate this experience.
  </p>
  <div
    className="tabs"
  >
    <p
      className="selected"
    >
      Human Profile
    </p>
    <p
      className=""
    >
      Pet Profile
    </p>
    <p
      className=""
    >
      Details
    </p>
  </div>
</div>
`;
